### PR TITLE
BLE: Add reference of Battery service.

### PR DESCRIPTION
**Pull request by pan-**
This page describes what the Bluetooth Battery Service is and what the BatteryService class do.

### CODE REVIEW

**Comment by pan-** [comment]
@sg- @AnotherButler Could you review ?

**Comment by AnotherButler** [comment]
@pan- Is this for 5.7, or is it applicable now?

**Comment by pan-** [comment]
It is applicable now.

File: docs/reference/api/connectivity/bluetooth/BatteryService.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
 ## BatteryService
 
-[Add description here.]
+It is often a requirement for devices operating on battery to report the battery charge level to the user.
+
+The <a href="https://www.bluetooth.org/docman/handlers/downloaddoc.ashx?doc_id=245138" target="_blank">Bluetooth Battery Service</a> defines how to expose a battery charge level through a BLE link. It allows a client - usually a smartphone application - of a device to read the current battery charge level and follow its evolution.
+
+The BatteryService class implements the Bluetooth Battery Service as defined by the Bluetooth body. Makers of BLE devices operating on battery can use the API to expose interoperably the charge level of their products.
```
Comment (sg-) [inline docs/reference/api/connectivity/bluetooth/BatteryService.md]: Bluetooth body or SIG (special interest group) as they refer to themselves as?

Comment (pan-) [reply docs/reference/api/connectivity/bluetooth/BatteryService.md]: I've replaced Bluetooth body by Bluetooth SIG.

**Comment by AnotherButler** [comment]
@pan- I'm having some trouble finding the correct API reference link. Is the doxygen for this in a features folder?

**Comment by pan-** [comment]
Yes, bless as a whole is in a feature folder.

**Pull request merged**